pkg/zhmcclient/lpar.go

```diff
@@ -42,12 +42,23 @@ type LparAPI interface {
 
 	ListNics(lparURI string) ([]string, int, *HmcError)
 	FetchAsciiConsoleURI(lparURI string, request *AsciiConsoleURIPayload) (*AsciiConsoleURIResponse, int, *HmcError)
+
+	GetEnergyDetailsforLPAR(cpcURI string, props *EnergyRequestPayload) (uint64, int, *HmcError)
```

Comment:
looks used different variable name `cpcURI` and `lparURI` here and L630.

Reply:
yes, we should keep consistency, and use `lparURI` here.